docs/conveyor/user-guide.md

```diff
@@ -0,0 +1,49 @@
+# Getting started
+## Preparation
+At first, you need to get some DAI and USDC tokens, the following steps show how to get these tokens on the Ethereum Ropsten testnet
+
+1. using [ropsten faucet](https://faucet.ropsten.be/) to get eth of ropsten testnet
+2. using [uniswap](https://app.uniswap.org/#/swap) to swap DAI and USDC, don't forget to connect to the ropsten test network in your Metamask
+![](../assets/uniswap_swap_network.jpg){width="673" height="463"}
+![](../assets/uniswap_swap_dai.jpg){width="673" height="463"}
+
+## Deposit
+In order to use conveyor, you need deposit your DAI and USDC to get the Wrapped token, such as gDAI or gUSDC
+![](../assets/gtoken_gdai.jpg){width="673" height="463"}
+
+1. Click `Deposit` and enter deposit amount
+![](../assets/gtoken_gdai_deposit.jpg){width="673" height="463"}
+
+2. Click `Deposit` again and use metamask to sign the transaction 
+![](../assets/gtoken_gdai_sign.jpg){width="673" height="463"}
+
+## Withdraw
+You can withdraw your deposit at any time
+1. Click `Withdraw` and enter amount
+![](../assets/gtoken_gdai_withdraw.jpg){width="673" height="463"}
+
+2. Click `Withdraw` again and use metamask to sign the transaction
+![](../assets/gtoken_gdai_withdraw_sign.jpg){width="673" height="463"}
+
+## Swap
+Select the token pair, the `From` represents token you own, `To` represents the token you want to exchange. Then enter the amount of `From` token, and the amount of `To` token will be estimated automatically.
```

Comment:
- the `From` represents the token you own
- and the amount of `To` tokens will be computed automatically for you, based on the liquidity of tokens and the current market price.